package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "JavaScript-Library-Project",
```

Comment:
Take a look at the [rules for the `name` field](https://docs.npmjs.com/files/package.json#name). You should not use uppercase:

> New packages must not have uppercase letters in the name.